src/components/trip/Note.js

```diff
@@ -0,0 +1,78 @@
+import React, { useState } from 'react'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import styled from 'styled-components/macro'
+import PropTypes from 'prop-types'
+
+Note.propTypes = {
+  title: PropTypes.string,
+  content: PropTypes.string,
+}
+
+export default function Note({ title, content }) {
+  const [fulltextVisible, setFulltextVisible] = useState(false)
+
+  return (
+    <NoteStyled onClick={showFulltext}>
+      <EntryType>
+        <FontAwesomeIcon className="icon" icon={['far', 'comment-dots']} />{' '}
+        Notiz
+      </EntryType>
+
+      <h2>{title}</h2>
+      {content && (
+        <p className={fulltextVisible ? 'fulltext' : ''}>{content}</p>
+      )}
```

Comment:
I think you don't need the ternary for the className here. How about rendering conditionally with `content && fulltextVisible && <p className="fulltext">{content}</p>`?